_use_cases/encryption-at-rest.markdown

```diff
@@ -0,0 +1,103 @@
+---
+name: Encryption At Rest
+---
+
+## Why
+
+Apache Kafka&#174; does not directly support any form of encryption for data stored within a broker. This means that the contents
+of records sent to Apache Kafka are stored in the clear. Anyone with sufficient privileges, such as a Kafka Administrator
+with file system access, is able to read the contents of the records.
+
+This present a challenge for an enterprise using Kafka to distribute confidential data or PII (Personally Identifiable
+Information).  Enterprises are often subject to confidential requirements coming from governmental bodies such as
+GDPR in the EU or LGPD in Brazil, industry standards such as HIPAA in the health domain or PCI/DSS in the payments domain,
+in addition to any in-house security and compliance requirements.
+
+The problem is made more complex if the enterprise has opted to utilise a Cloud Kafka Service as the confidential
+data is now residing in the clear on the file systems of the service provider.
+
+| ![image](../assets/encryption-at-rest_problem.png){:width="100%"} |
+|:-----------------------------------------------------------------:|
+|    *Problem: Plain text records readable by the Kafka Admins*     |
+
+### Isn't TLS sufficient?
+
+TLS encrypts the content as it traverses the wire.  It means that someone using a network sniffer cannot tap what is being
```

Comment:
> TLS encrypts the content as it traverses the wire.

Sounds odd to me I think it should be one of 

```suggestion
TLS encrypts the content as it traverses the network.  It means that someone using a network sniffer cannot tap what is being
```
or
```suggestion
TLS encrypts the content as it travels along the wire.  It means that someone using a network sniffer cannot tap what is being
```

Reply:
I simplified the sentence.